include/functions.h

```diff
@@ -922,7 +922,7 @@ void func_800BDC5C(SkelAnime* skelAnime, ActorAnimationEntry animation[], s32 in
 void Actor_Noop(Actor* actor, GlobalContext* globalCtx);
 void func_800BDFC0(GlobalContext* globalCtx, Gfx* dl);
 void func_800BE03C(GlobalContext* globalCtx, Gfx* dl);
-// void func_800BE0B8(UNK_TYPE1 param_1, UNK_TYPE1 param_2, UNK_TYPE1 param_3, UNK_TYPE1 param_4, UNK_TYPE4 param_5);
+Actor *func_800BE0B8(GlobalContext *globalCtx, Actor *inActor, s16 arg2, u8 arg3, f32 arg4);
```

Comment:
```suggestion
Actor* func_800BE0B8(GlobalContext* globalCtx, Actor* inActor, s16 arg2, u8 arg3, f32 arg4);
```
Did you run `./format.sh`, I think that's suppose to catch this? If it didn't work because `clang-format-11` isn't installed (it's an issues others have had and we have the solution, but haven't updated docs yet), we're more than happy to help in discord

Reply:
The formatter doesn't format header file :c

Reply:
I didn't run it on this because the whole file isn't really formatted anyway. I've added it now, and it's a 1,500 line diff. Almost 3x larger diff than the rest of the PR. :P